stories/forms/inputs/checkbox-group.story.js

```diff
@@ -75,3 +75,6 @@ storiesOf('CheckboxGroup', module)
       }}
     />
   ))
+  .add('with dropdown options', () => (
+    <CheckboxGroup input={inputProps} meta={{}} options={options} useDropdown />
```

Comment:
Prefer to be explicit here, based on rationale presented here: https://legacy.reactjs.org/docs/jsx-in-depth.html#props-default-to-true